pkgs/development/ocaml-modules/ppxlib/default.nix

```diff
@@ -1,21 +1,30 @@
 { lib, fetchFromGitHub, buildDunePackage, ocaml
-, version ? if lib.versionAtLeast ocaml.version "4.07" then "0.15.0" else "0.13.0"
+, version ? if lib.versionAtLeast ocaml.version "4.07" then "0.18.0" else "0.13.0"
```

Comment:
leave the default version at `0.15.0`, `>= 0.16.0` breaks a lot of packages currently.

Reply:
To be precise this currently breaks:

```
dune-release jackline libbap patdiff python37Packages.bap python38Packages.bap python39Packages.bap satysfi
```

Reply:
Fixed, thanks